hosts/tunnel.go

```diff
@@ -76,7 +79,7 @@ func checkDockerVersion(ctx context.Context, h *Host, clusterVersion string) err
 		return fmt.Errorf("Error while determining supported Docker version [%s]: %v", info.ServerVersion, err)
 	}
 
-	if !isvalid && !h.IgnoreDockerVersion {
+	if !isvalid {
```

Comment:
Need to revisit the entire if/else block. There is probably going to be only one if block when `&& !h.IgnoreDockerVersion` is removed.